website/docs/howdoesitwork.mdx

```diff
@@ -414,20 +414,20 @@ export namespace main {
       this.address = this.convertValues(source["address"], Address);
     }
 
-    convertValues(a: any, classs: any, asMap: boolean = false): any {
+    convertValues(a: any, class: any, asMap: boolean = false): any {
```

Comment:
I think this one was deliberate as it's a keyword

Reply:
> I think this one was deliberate as it's a keyword

I see, reverted this change.